opyplus/epgm/record.py

```diff
@@ -805,7 +805,7 @@ def to_json_data(self, model_name=None, named_keys=False):
         """
         data = {k: self.get_serialized_value(k, model_name=model_name) for k in self._data}
         if named_keys:
-            data = {self.get_field_descriptor(k).ref: self[k] for k, v in data.items()}
+            data = {self._table._dev_descriptor.get_extended_ref(k): v for k, v in data.items()}
```

Comment:
check that no function name_to_ref exists (in which case I would recommend to have one workflow)